android/build.gradle

```diff
@@ -1,25 +1,22 @@
 buildscript {
     repositories {
         google()
-        maven {
-            url "https://maven.google.com"
-        }
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0-beta03'
-        classpath 'com.google.gms:google-services:4.2.0'
```

Comment:
I would like to update the google services plugin on a different PR but unfortunately It is necessary to update together to be able to build 😬